### ISSUE TITLE
Steps to compile the project? Missing Readme

### ISSUE THREAD

**Issue by theedov** [tangem/tangem-app-ios-old#4456]
Hey,

Since the app is "open source", I think it'd be useful to create a readme with steps required to successfully compile the app.

Currently when you pull the repo it's missing files in BlockchainSdk/Resources/Generated as well as Localization, Colors, Assets, and some config files. I assume all these files will be generated after running some command.

**Comment by Newcomer365** [comment]
#2980 

There seems to be a solution here

**Comment by Elsewhere42** [comment]
I second this.

**Comment by tureck1y** [comment]
Indeed, building an application requires developer skills. For code generation you need to run a script from the root of the repository - bootstrap.sh.  We are planning some changes in the repository and will post instructions for building the application later.
